cmake/mapl-import.cmake.in

```diff
@@ -0,0 +1,3 @@
+list (APPEND CMAKE_MODULE_PATH @MAPL_FULL_INSTALL_DATA_DIR@/cmake)
```

Comment:
I don't recognize this variable.   Is it something that ecbuild defines for each project?